rec and auth: Implement an cors flag to set allowed origin in webserver (#17297)
PowerDNS/pdns: docs/upgrading.rst

5.0.0 to 5.1.0
--------------

Cross origin requests
^^^^^^^^^^^^^^^^^^^^^

> The embedded webserver now does not include a ``access-control-allow-origin: *`` header by default.

Reviewer: ```suggestion
The embedded webserver now no longer includes a ``access-control-allow-origin: *`` header by default.
```
